server/channels/web/oauth_test.go

```diff
@@ -272,7 +272,8 @@ func TestOAuthAccessToken(t *testing.T) {
 	require.NoError(t, err)
 	rurl, _ := url.Parse(redirect)
 
-	apiClient.Logout(context.Background())
+	_, err = apiClient.Logout(context.Background())
+	require.NoError(t, err)
```

Comment:
The check is failing with 

```
    oauth_test.go:276: 
        	Error Trace:	/mattermost/server/channels/web/oauth_test.go:276
        	Error:      	Received unexpected error:
        	            	json: cannot unmarshal number into Go value of type model.AppError
        	            	failed to decode JSON payload into AppError. Body: 404 page not found
        	            	
        	            	github.com/mattermost/mattermost/server/public/model.AppErrorFromJSON
        	            		/mattermost/server/public/model/utils.go:362
        	            	github.com/mattermost/mattermost/server/public/model.(*Client4).DoAPIRequestReader
        	            		/mattermost/server/public/model/client4.go:769
        	            	github.com/mattermost/mattermost/server/public/model.(*Client4).DoAPIRequest
        	            		/mattermost/server/public/model/client4.go:727
        	            	github.com/mattermost/mattermost/server/public/model.(*Client4).DoAPIPost
        	            		/mattermost/server/public/model/client4.go:699
        	            	github.com/mattermost/mattermost/server/public/model.(*Client4).Logout
        	            		/mattermost/server/public/model/client4.go:947
        	            	github.com/mattermost/mattermost/server/v8/channels/web.TestOAuthAccessToken
        	            		/mattermost/server/channels/web/oauth_test.go:275
        	            	testing.tRunner
        	            		/usr/local/go/src/testing/testing.go:1689
        	            	runtime.goexit
        	            		/usr/local/go/src/runtime/asm_amd64.s:1695
        	Test:       	TestOAuthAccessToken
```

Can you please take a look what is going on? Thanks!

Reply:
The test is still failing. Can you please take a look?